content/en/tracing/trace_collection/otel_instrumentation/go.md

```diff
@@ -23,10 +26,10 @@ The following OpenTelemetry features are implemented in the Datadog library as n
 
 | Feature                               | Support notes                       |
 |---------------------------------------|------------------------------------|
-| [OpenTelemetry Context propagation][1]         | [Datadog distributed header format][9] is used instead. |
+| [OpenTelemetry Context propagation][1]         | [W3C and Datadog header formats][9] are used. |
 | [Span processors][2]                  | Unsupported                                          |
 | [Span Exporters][3]                   | Unsupported                                            |
-| Trace/span [ID generators][4]         | ID generation is performed by `ddtrace`.           |
+| Trace/span [ID generators][4]         | [128-bit ID generation][12] is performed by `ddtrace`.           |
```

Comment:
Trace ID's and Span ID's have different sizes, so maybe we can reword it to something like this? I also substituted `ddtrace` with `tracing library` so we could copy/paste this description to other languages, if you think that's okay.

```suggestion
| Trace/span [ID generators][4]         | ID generation is performed by the tracing library, with support for [128-bit trace ID's][12].           |
```